code/create_interactome_files.R

```diff
@@ -1,289 +1,101 @@
-library(readxl)
-library(rjson)
-library(stringr)
-library(org.Hs.eg.db)
-library(xlsx)
-source("common.R")
+library(dplyr)
```

Comment:
Same here. Add a line of comments for some explanation of the script and authors